indico/modules/events/logs/client/js/components/EventLog.jsx

```diff
@@ -6,13 +6,43 @@
 // LICENSE file for more details.
 
 import React from 'react';
+import {useDispatch, useSelector} from 'react-redux';
+
+import {Param, Translate} from 'indico/react/i18n';
 
 import LogEntryList from '../containers/LogEntryList';
 import Toolbar from './Toolbar';
+import {fetchLogEntries, setMetadataQuery, setPage} from '../actions';
+
+function MetadataQueryMessage() {
+  const dispatch = useDispatch();
+  const resetURLFiltering = () => {
+    dispatch(setMetadataQuery({}));
+    dispatch(setPage(1));
+    dispatch(fetchLogEntries());
+  };
+
+  return (
+    <div className="highlight-message-box">
+      <span className="icon" />
+      <div className="message-text">
+        <Translate>
+          Log entries are currently filtered by URL.{' '}
+          <Param name="link" wrapper={<a onClick={resetURLFiltering} />}>
```

Comment:
`resetURLFiltering` works just fine. However, due to the fact this is combining react with server-side filtering, we will remain with the same URL query as before. In other words, we are only resetting this component's state. 

While this is not relevant, could be worth considering doing it all in the react app like https://github.com/indico/indico/pull/4503#discussion_r447708547 suggested or any workaround.

Reply:
`resetURLFiltering` now removes the arguments from the URL.